Makefile

```diff
@@ -28,14 +28,14 @@ endif
 
 GCC_FLAGS = -Wall
 
-MINGW_PREFIX = i686-w64-mingw32-
-MINCC = $(MINGW_PREFIX)gcc $(GCC_FLAGS)
+MINGW_PREFIX ?= i686-w64-mingw32-
+MINCC ?= $(MINGW_PREFIX)gcc
```

Comment:
I'm not sure why all the `?=` are necessary? `make MINGW_PREFIX= ...` already overrides `MINGW_PREFIX`?

Reply:
Indeed, fixed!